python/mspasspy/db/database.py

```diff
@@ -203,8 +203,21 @@ def set_database_schema(self, schema):
         """
         self.database_schema = schema
 
-    def read_data(self, object_id, mode='promiscuous', normalize=None, load_history=False, exclude_keys=None,
-                  collection='wf', data_tag=None, alg_name='read_data', alg_id='0', define_as_raw=False, retrieve_history_record=False):
+    def read_data(
+        self,
+        object_id,
+        mode="promiscuous",
+        normalize=None,
+        load_history=False,
+        exclude_keys=None,
+        collection="wf",
+        data_tag=None,
+        alg_name="read_data",
+        alg_id="0",
+        define_as_raw=False,
+        retrieve_history_record=False,
+        fill_value=None,
```

Comment:
We should probably add all three parameters from obspy here: `Stream.merge(method=0, fill_value=None, interpolation_samples=0)`. We can point user to obspy's detailed documentation here, which includes [this](https://docs.obspy.org/packages/autogen/obspy.core.trace.Trace.__add__.html#handling-overlaps) and [that](https://docs.obspy.org/packages/autogen/obspy.core.stream.Stream._cleanup.html#obspy.core.stream.Stream._cleanup).

Reply:
Agreed.